packages/mdc-drawer/adapter.ts

```diff
@@ -85,4 +85,14 @@ export interface MDCDrawerAdapter {
    * and restores focus to where it was prior to calling `trapFocus`.
    */
   releaseFocus(): void;
+
```

Comment:
Since this is a breaking change please move target branch (base branch) to `develop`.

Reply:
yes sure